docs/whats_new.rst

```diff
@@ -41,6 +41,7 @@ Enhancements
 - Adding :class:`braindecode.models.ATCNet` network for motor imagery (:gh:`429` by `Cedric Rommel`_)
 - Adding to :class:`braindecode.datasets.tuh.TUH` compatibility with version 3.0 of TUH dataset (:gh:`431` by `Mohammad Javad D`_, `Bruno Aristimunha`_, `Robin Tibor Schirrmeister`_, `Lukas Gemein`_, `Denis A. Engemann`_ and `Oskar Størmer`_)
 - Adding :class:`braindecode.models.DeepSleepNet` network for sleep staging (:gh:`417` by `Théo Gnassounou`_)
+- Adding :class:`braindecode.models.Conformer` network (:gh:`454` by `Yonghao Song`_)
```

Comment:
I would always (also the file name) call it EEGConformer for consistency.
Also: strictly speaking, the EEGConformer is not a "real" [Conformer ](https://arxiv.org/pdf/2005.08100.pdf).

Reply:
Thanks for your clarification! It's okay to call it EEGConformer. It's just for short before.

Reply:
Fixed @eeyhsong and @martinwimpff